### PR TITLE
docs: add matButtonModule to fix example

### PR DESCRIPTION
**Pull request by santoshyadav198613**

### CODE REVIEW

**Comment by devversion** [comment]
@santoshyadav198613 Is this ready for review? If not, can you please [convert this to a draft](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft).

If this is ready: I don't think any example in `cdk/a11y` relies on `MatButton`, or is the plan to add buttons here? (please explain in the commit message)

**Comment by santoshyadav198613** [comment]
Hi @devversion ,

The UI is broken as we are using mat-button in sample code template, but not importing it.
I need help to identify lint error as well.

**Comment by devversion** [comment]
@santoshyadav198613 Thanks for clarifying. Can you please share where you see the UI broken? I've looked in the docs site, and also can't see `mat-button` being used anywhere inside `cdk/a11y` examples.

**Comment by santoshyadav198613** [comment]
Hi @devversion ,
a11y is rolledback i did it to test, the actual issue is https://github.com/angular/components/blob/master/src/components-examples/cdk/text-field/text-field-autofill-monitor/text-field-autofill-monitor-example.html#L12 you can see we are using mat-raised-button, but on UI 
image

its normal image

**Comment by devversion** [comment]
Ahh. I see. You moved the changes over to `cdk/textfield` now. That looks good. Thx!

File: src/components-examples/cdk/text-field/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {TextFieldModule} from '@angular/cdk/text-field';
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
+import {MatButtonModule} from '@angular/material/Button';
```
Comment (devversion) [inline src/components-examples/cdk/text-field/index.ts:4]: This should be lower-case `button`.

```suggestion
import {MatButtonModule} from '@angular/material/button';
```

**Comment by santoshyadav198613** [comment]
OMG, how can I do that, thanks for pointing out, looks like I was too tired :)

**Pull request merged**